tests/js-test.spec.js

```diff
@@ -274,18 +273,15 @@ describe("GET operations", () => {
 				.then(() => {
 					done(new Error("Parameter should be an array of string or number"));
 				})
-				.catch((err) => {
-					expect(err).to.equal("Incorrect keys");
-					done();
```

Comment:
why did you delete error assert?

Reply:
the error message was previously inconsistent with all the others, this is more similar to what other methods do